pkgs/tools/text/mdbook-plantuml/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, fetchFromGitHub, stdenv, rustPlatform, darwin, pkg-config, openssl, libiconv }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "mdbook-plantuml";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "sytsereitsma";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1m53sp3k387injn6mwk2c6rkzw16b12m4j7q0p69fdb3fiqbkign";
+  };
+
+  cargoSha256 = "0xi14k86ym3rfz6901lmj444y814m7vp90bwsyjmcph3hdv6mjp0";
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ openssl.dev ]
+    ++ (lib.optionals stdenv.isDarwin
+      [ darwin.apple_sdk.frameworks.CoreServices ]);
```

```suggestion
  buildInputs = [ openssl.dev ]
    ++ lib.optionals stdenv.isDarwin [ CoreServices ];
```

Pleas inherit CoreServices from top-level.